### PR TITLE
revert feature: dont revert the input block config + confimation popup

### PR DESCRIPTION
**Pull request by fontanierh**

### CODE REVIEW

File: front/pages/w/[wId]/a/[aId]/runs/[runId]/index.tsx [hunk 1]
```diff
@@ -119,6 +123,7 @@ export default function AppRun({
 
     setIsLoading(false);
     setSavedRunId(run.run_id);
+    console.log(`Restored app ${app.sId} to previous version ${run.run_id}`);
```
Comment (fontanierh) [inline front/pages/w/[wId]/a/[aId]/runs/[runId]/index.tsx:126]: We get those on datadog right ?

Comment (spolu) [reply front/pages/w/[wId]/a/[aId]/runs/[runId]/index.tsx:126]: Nope you have to use `logger`

Comment (fontanierh) [reply front/pages/w/[wId]/a/[aId]/runs/[runId]/index.tsx]: done

File: front/pages/w/[wId]/a/[aId]/runs/[runId]/index.tsx [hunk 2]
```diff
@@ -119,6 +124,7 @@ export default function AppRun({
 
     setIsLoading(false);
     setSavedRunId(run.run_id);
+    logger.info(`Restored app ${app.sId} to previous version ${run.run_id}`);
```
Comment (spolu) [inline front/pages/w/[wId]/a/[aId]/runs/[runId]/index.tsx:127]: Let's structure it

```
logger.info(
{
  app: app.sId,
  run: run.runId
}, "Restored app to previous version");

Comment (fontanierh) [reply front/pages/w/[wId]/a/[aId]/runs/[runId]/index.tsx]: done

**Review by spolu — APPROVED** [review]
LGTM

**Pull request merged**